shared/domains/chelonia/internals.js

```diff
@@ -159,6 +196,13 @@ export default (sbp('sbp/selectors/register', {
     saferEval({
       // pass in globals that we want access to by default in the sandbox
       // note: you can undefine these by setting them to undefined in exposedGlobals
+      ...(typeof self !== 'undefined' && {
+        self: {
+          crypto: {
+            getRandomValues: (v) => self.crypto.getRandomValues(v)
+          }
+        }
+      }),
```

Comment:
Should this be exposed under `self`...? None of the other globals are. Maybe it should be directly included under `crypto`?

(And are we sure this is necessary here still?)